src/sentry/shared_integrations/client/proxy.py

```diff
@@ -51,6 +52,13 @@ def infer_org_integration(
     return org_integration_id
 
 
+def get_proxy_url() -> str:
+    control_address = getattr(settings, "SENTRY_CONTROL_ADDRESS", "")
+    control_address = control_address.rstrip("/")
+    proxy_base_path = PROXY_BASE_PATH.lstrip("/")
+    return f"{control_address}/{proxy_base_path}"
```

Comment:
You can also use `urllib.parse.urljoin` and it will take care of handling duplicate `/` and adding `/` as required.